test.py

```diff
@@ -48,7 +48,7 @@
 branch = args.branch
 noSync = args.noSync
 isWin = os.name == 'nt'
-DEBUG = True
+DEBUG = False # set this to True for debug output
```

Comment:
why don't you make this a command line argument? Then you don't need to change the source to see debugging output?

Reply:
..or use the already existing `verbose` flag for this purpose? :-)

Reply:
because that is really a debug variable, nobody but me wants to see all that output :)